# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
# allow to use Biopython/1.79-foss-2022a
diff -ru model-angelo-1.0.9/setup.py model-angelo-1.0.9_relax_requirements/setup.py
--- model-angelo-1.0.9/setup.py	2023-10-27 16:42:58.000000000 +0200
+++ model-angelo-1.0.9_relax_requirements/setup.py	2023-11-06 10:12:57.641982769 +0100
@@ -28,7 +28,7 @@
     install_requires=[
         "tqdm",
         "scipy",
-        "biopython>=1.81",
+        "biopython>=1.79",
         "einops",
         "matplotlib",
         "mrcfile",
